img/plan.sh

```diff
@@ -42,6 +42,14 @@ do_unpack() {
   return 0
 }
 
+do_prepare() {
+  pushd "${pkg_cache_path}" >/dev/null
+    go mod edit -replace github.com/containerd/containerd@3a3f0aac8819=github.com/containerd/containerd@814b795
```

Comment:
It might be useful to add a comment here why we're doing this, and why we're using `@814b795`.

Reply:
Added an explaination